src/enums/filingNames.ts

```diff
@@ -6,7 +6,7 @@ export enum FilingNames {
   CHANGE_OF_DIRECTORS = 'Director Change',
   CHANGE_OF_NAME = 'Legal Name Change',
   CHANGE_OF_REGISTRATION = 'Change of Registration',
-  CONVERSION = 'Conversion',
+  CONVERSION = 'Record Conversion',
```

Comment:
Not sure whether record conversion applies to benefit company conversions as well. Will have to check with Linda

Reply:
I don't know either. This is the title within Edit UI so for now it's probably a decent title in Filings UI.

For now it's for firms only: https://github.com/bcgov/business-filings-ui/blob/1fee2dcb0786d69f3bad9dfb6f188059d4e3671f/src/components/Dashboard/StaffNotation.vue#L69

Reply:
Also, there are tickets under this epic to verify/fix small issues like this:
https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/11224